src/components/lab-embed/embed-element/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { MutableRefObject, useRef, useEffect } from 'react'
```

Comment:
See the [lab embed in Learn](https://github.com/hashicorp/learn/tree/master/components/lab-embed), copied from there.